earth_enterprise/src/fusion/portableglobe/servers/portable_server.py

```diff
@@ -292,14 +292,18 @@ class QueryHandler(portable_server_base.BaseHandler):
   def get(self):
```

Comment:
Let's update the copyright notice to add `2019 Open GEE Contributors`. 
--> `Copyright 2017 Google Inc, 2019 Open GEE Contributors`

Reply:
All right, done!